composer.json

```diff
@@ -21,7 +21,13 @@
   "prefer-stable": true,
   "authors": [
     {"name": "Romans Malinovskis", "email": "romans@agiletoolkit.org", "homepage": "https://nearly.guru/" },
-    {"name": "Michael Voříšek", "homepage": "https://mvorisek.cz/"}
+    {"name": "Alain Belair"},
```

Comment:
@ibelar Is it ok to add you as a 2nd major contributor and do you have a homepage to add?

Reply:
oh - thanks - no home page.